src/yuzu/main.cpp

```diff
@@ -269,6 +270,15 @@ bool GMainWindow::CheckDarkMode() {
 #endif // __linux__
 }
 
+static void font_menu_items(QList<QAction*> the_actions) {
+    foreach (QAction* action, the_actions) {
```

Comment:
```suggestion
    for (QAction* action : the_actions) {
```

Since there's no need for the Qt-specific macro since C++11